### PR TITLE
[EcommerceFrameworkBundle] Mark classes as final/internal

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves partially #7189

### CODE REVIEW

File: bundles/EcommerceFrameworkBundle/EventListener/IndexUpdateListener.php [hunk 1]
```diff
@@ -20,7 +20,10 @@
 use Pimcore\Event\Model\DataObjectEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-class IndexUpdateListener implements EventSubscriberInterface
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/EventListener/IndexUpdateListener.php:24]: I think we should allow to modify that behavior as not every body whats to have index updates on object save.

File: bundles/EcommerceFrameworkBundle/Command/CleanupPendingOrdersCommand.php [hunk 2]
```diff
@@ -20,6 +20,9 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/CleanupPendingOrdersCommand.php:23]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/IndexService/BootstrapCommand.php [hunk 3]
```diff
@@ -24,6 +24,9 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/IndexService/BootstrapCommand.php:28]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/IndexService/EsSyncCommand.php [hunk 4]
```diff
@@ -22,6 +22,9 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/IndexService/EsSyncCommand.php:26]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/IndexService/ProcessPreparationQueueCommand.php [hunk 5]
```diff
@@ -24,6 +24,9 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/IndexService/ProcessPreparationQueueCommand.php:28]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/IndexService/ProcessUpdateIndexQueueCommand.php [hunk 6]
```diff
@@ -23,6 +23,9 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/IndexService/ProcessUpdateIndexQueueCommand.php:27]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/IndexService/ResetQueueCommand.php [hunk 7]
```diff
@@ -21,6 +21,9 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/IndexService/ResetQueueCommand.php:25]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/Voucher/CleanupReservationsCommand.php [hunk 8]
```diff
@@ -19,6 +19,9 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/Voucher/CleanupReservationsCommand.php:23]: do not mark as internal, but mark it as final

File: bundles/EcommerceFrameworkBundle/Command/Voucher/CleanupStatisticsCommand.php [hunk 9]
```diff
@@ -19,6 +19,9 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
```
Comment (fashxp) [inline bundles/EcommerceFrameworkBundle/Command/Voucher/CleanupStatisticsCommand.php:23]: do not mark as internal, but mark it as final

**Review by fashxp — CHANGES_REQUESTED** [review]

File: bundles/EcommerceFrameworkBundle/EventListener/IndexUpdateListener.php (hunk 1 above)
Comment (dvesh3) [reply bundles/EcommerceFrameworkBundle/EventListener/IndexUpdateListener.php]: Reverted

**Review by fashxp — APPROVED** [review]

**Pull request merged**